quint/src/quintError.ts

```diff
@@ -61,8 +63,8 @@ export type ErrorCode =
   | 'QNT406'
   /* QNT407: Cannot import self */
   | 'QNT407'
-  /* QNT099: Found cyclic definitions */
-  | 'QNT099'
+  /* QNT500: Unitialized constant */
+  | 'QNT500'
```

Comment:
Just curious, do we have a numbering scheme for these?

Reply:
Not really, I used `5**`, as REPL and simulator are at phase 5.